src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -0,0 +1,335 @@
+        _autoDelayChooser.addOption("2", 2);
+        _autoDelayChooser.addOption("3", 3);
+        _autoDelayChooser.addOption("4", 4);
+        _autoDelayChooser.addOption("5", 5);
+        autonomousLayout.add("Delay Options", _autoDelayChooser).withPosition(0, 0).withSize(1, 1).withWidget(BuiltInWidgets.kSplitButtonChooser);
+
+        // Autonomous start position chooser setup
+        // _autoStartPositionChooser = new SendableChooser<>();
+        // _autoStartPositionChooser.setDefaultOption("Middle",
+        // DrivePosition.MiddleStart);
+        // _autoStartPositionChooser.addOption("Substation Side",
+        // DrivePosition.SubstationStart);
+        // _autoStartPositionChooser.addOption("Wall Side", DrivePosition.WallStart);
+        // autonomousLayout.add("Start Position", _autoStartPositionChooser)
+        // .withPosition(0, 1)
+        // .withSize(1, 1)
+        // .withWidget(BuiltInWidgets.kComboBoxChooser);
+
+        var outerLayout = SettingsTab.getLayout("Settings", BuiltInLayouts.kGrid).withPosition(0, 0).withSize(40, 10).withProperties(Map.of("Number of columns", 6, "Number of rows", 1, "Label position", "TOP"));
+
+        var driveSettings = outerLayout.getLayout("Drive Subsystem", BuiltInLayouts.kList).withPosition(0, 0).withSize(8, 10).withProperties(Map.of("label position", "LEFT"));
+        _flOffset = driveSettings.add("FL Offset", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _frOffset = driveSettings.add("FR Offset", 0.0).withPosition(2, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _blOffset = driveSettings.add("BL Offset", 0.0).withPosition(0, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _brOffset = driveSettings.add("BR Offset", 0.0).withPosition(2, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var IntakeSetting = outerLayout.getLayout("Intake", BuiltInLayouts.kList).withPosition(1, 0).withSize(7, 10).withProperties(Map.of("label position", "LEFT"));
+        _IntakeInSpeed  = IntakeSetting.add("Intake In Speed", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).withPosition(0, 0).getEntry();
+        _IntakeOutSpeed = IntakeSetting.add("Intake Out Speed", 0.0).withWidget(BuiltInWidgets.kTextView).withPosition(0, 1).getEntry();
+
+        var climbSetting = outerLayout.getLayout("Climber", BuiltInLayouts.kList).withPosition(2, 0).withProperties(Map.of("label position", "LEFT")).withSize(7, 10);
+        _climberleftOffset  = climbSetting.add("Left Offset", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _climberRightOffset = climbSetting.add("Right Offset", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _climberMinHeight   = climbSetting.add("Arm Minimum Height", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _climberMaxHeight   = climbSetting.add("Arm Maximum Height", 0.0).withPosition(2, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
```

Comment:
1. Updated tab name
2. Removed `withSize` and `withPosition`
3. Updated variable names

```suggestion
        var climbSettings = outerLayout.getLayout("Climb", BuiltInLayouts.kList).withPosition(2, 0).withProperties(Map.of("label position", "LEFT"));
        _climbLeftOffset  = climbSettings.add("Left Offset", Constants.Climb.LEFT_ZERO_OFFSET).withWidget(BuiltInWidgets.kTextView).getEntry();
        _climbRightOffset = climbSettings.add("Right Offset", Constants.Climb.RIGHT_ZERO_OFFSET).withWidget(BuiltInWidgets.kTextView).getEntry();
        _climbMinHeight   = climbSettings.add("Arm Minimum Height", Constants.Climb.MIN_EXTENSION).withWidget(BuiltInWidgets.kTextView).getEntry();
        _climbMaxHeight   = climbSettings.add("Arm Maximum Height", Constants.Climb.MAX_EXTENSION).withWidget(BuiltInWidgets.kTextView).getEntry();
```